package.json

```diff
@@ -8,7 +8,7 @@
 	},
 	"types": "./lib/index.d.ts",
 	"dependencies": {
-		"ardrive-core-js": "../ardrive-core-js_PE-463_download_folder",
+		"ardrive-core-js": "../ardrive-core-js_PE-780_download_custom_folder_name",
```

Comment:
TODO: Update

Reply:
The core version is currently set to an alpha version as in the PR base.

Closing this comment and adding at the base PR